### ISSUE TITLE
force:source:manifest:create --fromorg creates manifest with missing namespace on CMT records and layouts

### ISSUE THREAD

**Issue by kalius** [forcedotcom/cli#1697]
### Summary

When using force:source:manifest:create --fromorg, namespace is missing on managed Custom Metadata Type records and Layouts
Side effect: If there are two CMT records with same developername but different QualifiedApiName (one managed and other unmanaged), only one is present in package.xml

### Steps To Reproduce:

Step 1: Install any managed package with public (not protected) Custom Metadata Type record and managed Layout
Step 2: Run ```sfdx force:source:manifest:create --fromorg org_alias -n package.xml -c managed```

### Expected result

```
<types>
	<members>namespace__Object.namespace__Layout</members>
	<name>Layout</name>
</types>
<types>
	<members>namespace__Object.namespace__Record</members>
	<name>CustomMetadata</name>
</types>
```

### Actual result

```
<types>
	<members>namespace__Object.Layout</members>
	<name>Layout</name>
</types>
<types>
	<members>namespace__Object.Record</members>
	<name>CustomMetadata</name>
</types>
```

### System Information

Tried in bash and cmd.

sfdx-cli/7.166.1 win32-x64 node-v16.15.1

**Comment by jh480** [comment]
Applies also to QuickAction & GlobalValueSetTranslation. Similar behaviour with Metadata API's _listMetdata_ (ant).

**Comment by WillieRuemmele** [comment]
Hi @jh480 @kalius - do you know of any "managed package with public (not protected) Custom Metadata Type record and managed Layout" that I can install into a scratch org to test this out?

**Comment by jh480** [comment]
@WillieRuemmele, maybe FinServ (Group Builder Field Sets/ Job Flow Settings/ Use Standard Triggers)? https://www.salesforce.com/form/signup/financial-services-cloud-trial/

Though afaik protection is not relevant, problem relates to all managed Custom Metadata.

**Comment by mshanemc** [comment]
Unfortunately, the fromOrg feature is getting its information from the same place (metadataApi#list)

https://github.com/forcedotcom/source-deploy-retrieve/blob/b1d5ec90e358e65cb454358f0bb785cec42bce9a/src/resolve/connectionResolver.ts#L96

I'm going to mark this as "owned by another team".  You should open a Case to help get attention, and preferably limit the repro to the metadata api `list` command so that's it's clearly an API problem and the CLI doing something.

**Comment by kalius** [comment]
@WillieRuemmele, yes I have one public package with all available stuff for testing. I can share the link if you want to look at it.

yeah, probably it is really api problem..

**Comment by jh480** [comment]
Apparently, this is "working as designed".
Here's an idea about it: https://ideas.salesforce.com/s/idea/a0B8W00000NX7GOUA1/list-managed-metadata-for-layout-quickaction-etc-with-correct-namespace-prefix

**Comment by mshanemc** [comment]
[W-15587165](https://gus.lightning.force.com/lightning/r/a07EE00001prEgpYAE/view)
